esbuildBabelPluginIstanbul.ts

```diff
@@ -0,0 +1,38 @@
+import { transformFileAsync } from '@babel/core';
```

Comment:
We cannot use `esbuild-plugin-istanbul` because it sometimes not emitting instrumentation code related to JSX.